File: src/main/java/com/my/admin/app/commerce/model/api/response/BasicCashResponse.java
package com.my.admin.app.commerce.model.api.response;

import com.fasterxml.jackson.annotation.JsonInclude;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

@Data
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class BasicCashResponse {
    @ApiModelProperty(value = "Cash Id", example = "1")
    private String id;

    @ApiModelProperty(value = "Turn value for cash")
    private TurnResponse turn;

    @ApiModelProperty(value = "Cash Status description", example = "Active")
    private CashStatusResponse status;

    @ApiModelProperty(value = "User value of cash", example = "1")
    private UserResponse user;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/api/response/BasicCashResponse.kt
package com.bk.reactive.app.commerce.admin.my.model.api.response

import com.fasterxml.jackson.annotation.JsonInclude
import io.swagger.annotations.ApiModelProperty

@JsonInclude(JsonInclude.Include.NON_EMPTY)
open class BasicCashResponse(
    @ApiModelProperty(value = "Cash Id", example = "1")
    val id: String,

    @ApiModelProperty(value = "Turn value for cash")
    val turn: TurnResponse,

    @ApiModelProperty(value = "Cash Status description", example = "Active")
    val status: CashStatusResponse,

    @ApiModelProperty(value = "User value of cash", example = "1")
    val user: UserResponse
)